en.lproj/Localizable.strings

```diff
@@ -59,6 +59,7 @@
 "NICK_ADD" = "Message a User";
 "NICK_ADD_DONE" = "Message";
 "NICK_ADD_TITLE" = "Message User";
+"NETWORK_ADD_TITLE" = "Add Network";
```

Comment:
Can we move this down to the `NETWORK_ADD` section below (`/* Add Network */`) so that all the messages for the same screen is in the same place.

Reply:
Sure, its moved!